allenact_plugins/manipulathor_plugin/manipulathor_environment.py

```diff
@@ -292,8 +419,8 @@ def step(
 
         if self.simplify_physics:
             action_dict["simplifyOPhysics"] = True
-        if action in ["PickUpMidLevel", "DoneMidLevel"]:
-            if action == "PickUpMidLevel":
+        if action in [PICKUP, DONE]:
```

Comment:
It's a bit weird to me that all this step control is happening in the `ManipulaTHOR` environment class. E.g.  that `RotateRight` corresponds to a 45 degree rotation should be something that's specified by the task and not the environment. Perhaps not something to worry about now though as this is the current design.